projects/zircon-gamma/index.js

```diff
@@ -22,6 +38,9 @@ async function tvl(_, _b, _cb, { api, }) {
 module.exports = {
   misrepresentedTokens: true,
   moonriver: {
-    tvl: sdk.util.sumChainTvls([tvl, uniTvl]),
+    tvl: sdk.util.sumChainTvls([tvl, movrUniTvl]),
   },
+  bsc: {
+    tvl: sdk.util.sumChainTvls([bscUniTvl]),
```

Comment:
Missing bscTvl?

Reply:
Unfortunately, there are certain restrictions on eth_getlogs when using the Binance Smart Chain (BSC). I'm wondering whether it would still be feasible to use the Graph for BSC, or if you have any alternative suggestions that we could explore?

Reply:
https://github.com/BNB-Chain-Support/bnb-chain.github.io/blob/master/docs/rpc.md

Reply:
@VittoriDavide graph could work, we cache the log messages. so it is painful the first call, or we can hard code the data/addresses till now and use logs to fetch new pools (by setting block number to todays)

Reply:
I think it works if we use bsc archival node, so fixed it here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/5422

Reply:
hey @g1nt0ki! The error of eth_getlogs is still occurring in the branch you merged

Reply:
@VittoriDavide yeah, thats because we have not added archival node to github config, it uses default public node